.travis.yml

```diff
@@ -21,14 +18,43 @@ env:
     - GH_REF: github.com/paulgibbs/behat-wordpress-extension
     - secure: RDArf/FuQFEz0O5n9NnWq9efe8tOH8eQIH28rNhSGA1lZ57zo2+3fU6QRvZrvW4fb7bIN6cvZSlOnsO3RYrf/UUwPSItjn9Jiia1wo70iqvdWw9VRtZ7YWysHkxi3tfEhIoBkHJMXJ7xhPrGg/qAevuoXWAkuocRUg370Fpe1oyMT6FC9brCsfgt6kJeCDEjeRrxUVCaozpTpX8/K/Glfu2lU8jiF6U5GvoT8+fzVsAgs/zJz9CoJcAzhWl/b2UrF5H2pqd6LoNad93NM5c5sG69E3VtuxO0ahECk1d2FttL6cEeSXM81ZRO4tRXfUfkvLu43kRVdNSYdRlD+XjQ2s+GcNvVMvmS/kun19rOxxlf+rgbYMcVoENC/DsSu0Qvk/y++POJHNLVwPECAviHK1qEgvdhMMsN87lh/op0aDLB1mKIYj4DbF5gHt4d16QfAJVTx8SeYbqRJrbHHXcRG3HHWwPp5mxWUoknvAkZgWRqZgByXdW1oB81qqqDZ42L1iGCmjufDYF6gw9dzaUERJzO/EmgS6dqeflUJIYxjn8bNrrCwEvYOTFNElHg78M/hUs046i4Ui9xd1kJUhEKduX9HoLalfzTwF+qAz+lEHOGXkIs/1hWB6UhacggnWPj7/GPXi4OgWwPPgJuHJ+MfmO/frXx3h/mYTx8OzMv8NU=
 
-before_script:
+before_install:
+  - sudo apt-get update > /dev/null
+  - composer self-update
+
+  # Install Apache
+  - sudo apt-get install -y --force-yes apache2 libapache2-mod-php5 php5-curl php5-mysql php5-intl
```

Comment:
I have no strong feelings about the PHP version, but it would be good to ensure all drivers are tested (presumably we can just call `behat` once for each driver).

If WordPress (and so WP-AP)I driver wouldn't work on PHP 7.1, presumably WP-Cli wouldn't either? After all, that just invokes WordPress' functions?